### ISSUE TITLE
AttributeError: 'unicode' object has no attribute 'items'

### ISSUE THREAD

**Issue by AlexOugh** [puresec/aws-parsecf#10]
'isinstance(\<variable\>, unicode) needs to be added when 'isinstance(\<variable\>, str) is used in 'conditions.py', 'functions.py' and 'parse.py'

**Comment by odedniv** [comment]
Hi @AlexOugh, thanks for the bug report.

Do you think you can submit a PR to fix this? That would ensure the fix works with your use case.

**Comment by AlexOugh** [comment]
Sure, but I don't have a permission to create a new branch.

**Comment by odedniv** [comment]
That's true, you should fork the repository and create a branch in your forked repo. Let me know if you need any help.

**Comment by odedniv** [comment]
@AlexOugh apparently you can use `isinstance(<variable>, basestring)` in Python 2.x so it works for both `str` and `unicode` types. Unfortunately `basestring` is not available in Python 3.x.
